www/blog/2022-11-06-announcing-trpc-10.mdx

```diff
@@ -0,0 +1,89 @@
+---
+slug: announcing-trpc-10
+title: Announcing tRPC 10
+author: Alex / KATT 🐱
+author_title: Creator of tRPC
+author_url: https://twitter.com/alexdotjs
+author_image_url: https://avatars1.githubusercontent.com/u/459267?s=460&v=4
+---
+
+import { GithubSponsorButton } from '@site/src/components/GithubSponsorButton';
+
+## Introduction
+
+tRPC's mission from day one has been to provide you with an easy way of moving fast and not breaking nothing whilst building full-stack applications.
+
+Since we released our last major version in August 2021, we've gotten over 13,500+ more stars on GitHub, a [Discord-community](https://trpc.io/discord) with over 2,000 members, thousands more applications built using tRPC, a growing ecosystem, and countless community members contributing and providing valuable feedback.
+
+Today, **we're launching tRPC 10**. It is already used in production and can safely be used for in your next, or current, TypeScript project.
+
+Get up and running with the [example applications](https://trpc.io/docs/v10/awesome-trpc#-starting-points-example-projects-etc) to get a feel of tRPC 10, or see the [guide for migrating from tRPC 9 to tRPC 10](https://trpc.io/docs/v10/migrate-from-v9-to-v10).
+
+## Overview of changes
+
+tRPC 10 is our biggest release ever. This is the first time we have made any fundamental changes to the structure of tRPC, yet these changes unlock new possibilities for cross-functional teams working on cutting-edge applications.
+
+tRPC 10 includes:
+
+- **Massively Improved TypeScript Performance:**
+- **Improved API Experience**
+  - Calling functions instead of invoking strings
+  - CMD+clicking and refactoring from the frontend
+  - Easier to infer types
+- **Powerful Backend Framework**
+  - Reusable middlewares
+  - Reusable base procedures
```

Comment:
I think these sections should be sorted in order of importance. Yes, the performance win was great but it's not really the core of what v10 was about. The changes you describe here also only make sense to people who have used tRPC v9. Here's the order I would use (from greatest to least importance):
- blurs the lines between your frontend and backend (improved API experience)
  - cmd+click (jump to definition) from frontend queries to the backend functions
- powerful backend framework emphasizing reusability (or composition) so you never have to repeat yourself
- massively improved TypeScript performance

The ones I removed from above aren't easy to grasp through text unless you've already used tRPC v10, in which case they should probably be expanded upon in the individual sections.

Reply:
Might be worth adding an additional second bullet point in the first item: "Fn+F2 style renaming of queries and mutations" right after the with the "cmd+click".